### DISCUSSION TITLE
Change release name format

### DISCUSSION THREAD

**Discussion by Mitia99** [MustardChef/WSABuilds#111] [Ideas]
Dear dev,

Thanks for updates,

Could you consider changing build release name format?

I mean removing "Windows Subsystem for Android: Windows 11 x64".

I think that "Windows_11_WSA_2305.40000.6.0_X64" is enough. If not, adding a guide in readme for which version to download will help new users.

My issue is that I track updates on Github android app and release version is not displayed completely.

Thanks,

![AddText_06-23-10.16.05_edit_2044254073646.jpg](https://github.com/MustardChef/WSABuilds/assets/56766778/6f3fabde-0245-4b5e-89ce-c2ea6031e390)

**Comment by MustardChef** [comment]
Thank you for your suggestions, I will keep this in mind in the future. 

If you stuggle to find a release in the meantime, check the latest release button in the[ Download section of the Readme.md](https://github.com/MustardChef/WSABuilds#downloads) or check [Old Builds list](https://github.com/MustardChef/WSABuilds/blob/master/OldBuilds.md) for previous releases. These have been setup as a temporary measure, while I get around restructuring the releases format/structure in the future.
